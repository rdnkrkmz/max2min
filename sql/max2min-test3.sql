-- 3. case: char, varchar, text
select max_to_min(val, ' => ') from (values('a'),('b'),('c'),('d'),('e'),('f'), (null),('g')) t(val);
