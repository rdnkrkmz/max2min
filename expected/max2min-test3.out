-- 3. case: char, varchar, text
select max_to_min(val, ' => ') from (values('a'),('b'),('c'),('d'),('e'),('f'), (null),('g')) t(val);
 max_to_min 
------------
 g => a
(1 row)

